dev/cognitiveservices/data-plane/Language/analyzetext-authoring.json

```diff
@@ -1882,6 +2385,41 @@
       },
       "x-ms-client-name": "ConfusionMatrixRow"
     },
+    "TextAnalysisAuthoringCopyProjectOptions": {
+      "description": "Represents the options for copying an existing project to another Azure resource.",
+      "required": [
+        "targetProjectName",
+        "accessToken",
+        "expirationDateTime",
+        "targetSubscriptionId",
+        "targetResoureRegion"
+      ],
+      "type": "object",
+      "properties": {
+        "targetProjectName": {
+          "description": "The project name to be copied-into.",
+          "type": "string"
+        },
+        "accessToken": {
+          "description": "The access token.",
+          "type": "string"
+        },
+        "expirationDateTime": {
```

Comment:
Nit: API Stewardship Board is debating on exact naming for date-times here, but if this includes a time I recommend `expiresAt` instead.